mne/decoding/base.py

```diff
@@ -454,18 +486,47 @@ def fit(self, X, y, **fit_params):
 
     @property
     def filters_(self):
-        if hasattr(self.model, "coef_"):
+        if hasattr(self.model_, "coef_"):
             # Standard Linear Model
-            filters = self.model.coef_
-        elif hasattr(self.model.best_estimator_, "coef_"):
+            filters = self.model_.coef_
+        elif hasattr(self.model_, "estimators_"):
+            # Linear model with OneVsRestClassifier
+            filters = np.vstack([est.coef_ for est in self.model_.estimators_])
+        elif hasattr(self.model_, "best_estimator_") and hasattr(
+            self.model_.best_estimator_, "coef_"
+        ):
             # Linear Model with GridSearchCV
-            filters = self.model.best_estimator_.coef_
+            filters = self.model_.best_estimator_.coef_
         else:
             raise ValueError("model does not have a `coef_` attribute.")
         if filters.ndim == 2 and filters.shape[0] == 1:
             filters = filters[0]
         return filters
 
+    # XXX Remove this property after 'model' warning cycle
+    @property
+    def model(self):
+        if "model_" in self.__dict__:
+            depr_message = (
+                "Starting with mne-python v1.13 'model' attribute "
+                "of LinearModel will not be fitted, "
+                "please use 'model_' instead"
+            )
+            warn(depr_message, FutureWarning)
+            return self.model_
```

Comment:
I wonder if, instead of doing this, we can consider it a bugfix and just leave `self.model` as the unfitted model (or None) and work with `self.model_` in all of our code. This behavior change depending on whether or not the model has been fit seems a bit complicated and has the risk to surprise users (too magical that what gets returned changes)

Reply:
It might, but we are allowed to break code for bugfixes. Especially if the corresponding fix to their code is simple (and here it's just adding a `_`) then we should be good to go. In the message we could even tell them "to be compatible with all MNE-Python versions you can do `getattr(clf, "model_", getattr(clf, "model"))`" as this will work on all versions of MNE-Python